productivity-suite/app/legacy-app/index.html

```diff
@@ -161,6 +161,7 @@
       }
 
       piercing-fragment-host {
+        display: block;
```

Comment:
I've moved the display block here because it needs to always apply, otherwise we have different widths for the fragments depending on whether seams are on or not